CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.39.0
```

Comment:
can you add this:

* Fix an edge case that can cause task failed to start when using container ordering success condition [#2404](https://github.com/aws/amazon-ecs-agent/pull/2404).

Reply:
Added.